Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp

```diff
@@ -924,10 +573,8 @@ std::unique_ptr<WebGLRenderingContextBase> WebGLRenderingContextBase::create(Can
     RefPtr<GraphicsContextGL> context;
     if (hostWindow)
         context = hostWindow->createGraphicsContextGL(attributes);
-    else {
-        // FIXME: OffscreenCanvas does not support GPU process.
-        context = createWebProcessGraphicsContextGL(attributes);
-    }
+    // FIXME: OffscreenCanvas does not support GPU process and ANGLE does not support 
```

Comment:
@zdobersek since review, I had to disable offscreen canvas. It won't work in-process with ANGLE and we haven't yet fixed needed stuff for it to work with GPUP. 
So to get offscreen canvas back, we'll work implementing it in GPUP. Still ok?

Reply:
OK.